src/controllers/webhookController.ts

```
@@ -14,6 +14,7 @@ function setFiatWebhookDeprecationHeaders(res: Response): void {
 }
 import { Request, Response, NextFunction } from "express";
 import crypto from "crypto";
+import { generateId } from '../utils/idGenerator';
```

ESLint

[error] Replace `'../utils/idGenerator'` with `"../utils/idGenerator"`

(prettier/prettier)